voicevox_engine/tts_pipeline/tts_engine.py

```diff
@@ -174,6 +174,23 @@ def apply_pitch_scale(moras: list[Mora], query: AudioQuery) -> list[Mora]:
     return moras
 
 
+def apply_pause_length(moras: list[Mora], query: AudioQuery) -> list[Mora]:
+    """モーラ系列へ音声合成用のクエリがもつ無音時間(絶対値)（`pauseLength`）を適用する"""
```

Comment:
周りと合わせたドキュメントの追加ありがとうございます！

こっちはおそらくこの説明で大丈夫そう
```suggestion
    """モーラ系列へ音声合成用のクエリがもつ無音時間（`pauseLength`）を適用する"""
```